app/assets/javascripts/components/activity/recent_activity_handler.cjsx

```diff
@@ -10,16 +10,21 @@ RecentEditsHandler = require './recent_edits_handler'
 RecentActivityHandler = React.createClass(
   displayName: 'RecentActivityHandler'
   render: ->
+
+    dIYEligible = I18n.t('recent_activity.did_you_know_eligible')
```

Comment:
This variable name is misleading. You meant `dYKEligible`, I assume.

In my opinion, a better option more generally here would be to just put the I18n calls directly into the markup, instead of storing them in these variables first. With a long string specified in the file itself, having a variable for the message makes it easier to see what's going on with the html without having the content there to confuse things, but the i18n message keys do just as good or better a job than variable names at concisely indicating what's going on.